storage/mysql/storage.sql

```diff
@@ -89,6 +93,8 @@ CREATE TABLE IF NOT EXISTS SequencedLeafData(
 
 CREATE TABLE IF NOT EXISTS Unsequenced(
   TreeId               INTEGER NOT NULL,
+  -- Note that this is a simple SHA256 hash of the raw data used to detect corruption in transit.
+  -- It is not the leaf hash output of the treehasher used by the log.
```

Comment:
At this point, how much of a pain would it be to rename the column rather than documenting around it?  (E.g. to something like PayloadHash or RawLeafHash?  Or maybe rename the 2 above to MerkleLeafHash)